lib/libc/sys/procctl.2

```diff
@@ -712,6 +712,50 @@ following statuses:
 The status is or-ed with the
 .Va PROC_KPTI_STATUS_ACTIVE
 in case KPTI is active for the current address space of the process.
+.Sh CHERI SPECIFIC REQUESTS
+.Bl -tag -width PROC_CHERI_REVOKE_STATUS
+.It Dv PROC_CHERI_REVOKE_CTL
+Controls the runtime hint to enable or disable heap temporal safety in
+the program images created
+by
+.Xr execve 2
+in the specified process or its descendants that did not changed
```

Comment:
```suggestion
in the specified process or its descendants that did not change
```

Reply:
oops, that's a cut and paste-o from upstream